config/application.rb

```diff
@@ -28,6 +28,10 @@ class Application < Rails::Application
         methods: [:get, :post, :options, :delete, :put, :patch]
       end
     end
+
+    Wor::Paginate.configure do |config|
+      config.formatter = Wor::Paginate::Formatters::PankoFormatter
+    end
```

Comment:
Move this config to wor-paginate initializer 


* Then you can run `rails generate wor:paginate:install` to create the initializer for configuration details, including default formatter class, page and limit params and default page number.